_posts/2016-12-02-recurse-center-week-9.md

```diff
@@ -0,0 +1,167 @@
+
+So instead I deny it.
+
+It's not that I'm **afraid**.  
+I just **don't care**  ¯\\\_(ツ)\_/¯.
+
+Since starting to ponder this I've been better at noticing it and acting anyway.
+
+### Social
+
+#### **Chill time with friends**
+
+Despite my apathy, it's been really nice getting opportunities to hang out with people outside of RC.
+
+On Monday, I finally got to see a friend from my Microsoft Intern days.  
+On Friday, I went with some other RCers to a board games cafe.  
+
+Hearts for low-key socializing! ❤️
+
+#### **Potlucks and parties**
+
+Thanksgiving caused RC to be closed for the majority of the week. A bunch of us aren't from New York / didn't have plans, so @ebb-tide hosted a Thanksgiving potluck.
+
+It brought me back to my dinner party days in Vancouver.  
+Tables full of food and drink. Couches to snuggle up on. Wonderful humans to talk to.
+
+Was so full of gratitude and happiness the whole evening. Gah! 😍
+
+Wednesday there was a smaller gathering. It was nice being with close pals and also meeting some people from outside the RC community. 🙂
+
+Such a lovely mélange of socializing this week!
+
+### Intellectual
+
+#### **Building communities**
+
+I had a lot of great conversations throughout the week about *"What comes after RC?"*.
+
+Another idea that's gotten me excited is starting a coliving space. I spent most of Saturday:
+
+- Asking people about past experiences with coops.
+- Researching existing coops/coliving spaces.
+- Aggregating lists of resources.
+- Debating ideas with friends.
+
+I've started looking for spaces to visit or apply to. One consistent piece of advice was to give it a try before diving in.
+
+Nevertheless, questions around community building have been fascinating me and it seems like a stream of enthusiasm worth following.
+
+#### **How to spend my time?**
+
+Another question I've been pondering is *"How do I want to spend my time?"*.
+
+My one approach to making time decisions, is *"What are my goals? Is this the most impactful way to move towards them?"*.
+It's interesting how this changes the way I spend time.
+
+**One example:**
+
+I used to read a lot online. In fact, [EpubPress](https://epub.press) was created to help me manage my intake of reading material.
+
+- Quora
+- HackerNews
+- Newspapers
+- etc.
+
+I've since realized the majority of what I read isn't immediately useful to me.
+
+> "How we scale @ {insert_startup}"  
+> Ooo! How DO they scale? 😱
+
+*I don't know... but why does it matter? I see no need to create a hyper growth system any time soon. If I ever do, I'm sure there's better resources than something from HackerNews.*
+
+> "Introducing {insert_generic_word}. A simple, elegant and intuitive JS framework"  
+> Ooo, I write Javascript! I haz need for this 😱!
+
+*No you don't, Harold. You're in no need of a framework... in fact, nobody writing Javascript needs another framework.*
+
+I've generally found this pushing my time towards more fulfilling activities.
```

Comment:
I've let my unread news grow out of control, and I become discouraged from looking nowadays because there's just too much. 

I use a filter for HackerNews stories, so only high ranking stories show in my feed: https://edavis.github.io/hnrss/. I'll skim over those occasionally but always thinking "If you won't use it, don't read it". Being mindful helps me avoid the click bait stuff.

When I do read stuff, it's because it answers a question I'm having at the moment. I search it out rather then hoping it finds me. 

Also getting news via friend- recommendations is huge: (Eg. "Hey X, know any good articles I can read about Y?").

Ya, that probably sums it up. :)